File: 이건희/level1/2016년.py
def solution(a, b):
    week = ['SUN','MON','TUE','WED','THU','FRI','SAT']
    month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    day = 4
    for i in range(a - 1):
        day += month[i]
    day += b
    day %= 7
    return week[day]

File: 이훈일/Programmers/Level1/2016년.java
import java.util.HashMap;

class Solution {
    public String solution(int a, int b) {
        String[] week = new String[]{"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
        HashMap<Integer, Integer> map = new HashMap<>(){{
            put(1,31);
            put(2,29);
            put(3,31);
            put(4,30);
            put(5,31);
            put(6,30);
            put(7,31);
            put(8,31);
            put(9,30);
            put(10,31);
            put(11,30);
            
        }};
        int sum = 0;
        for(int i = 1; i < a; i++) {
            sum += map.get(i);
        }
        return week[(sum + b -1) % 7];
    }
}
